src/classes/groundplaces.ts

```diff
@@ -27,23 +40,34 @@ export class GroundPlaces {
   /**
    * @description Create the stopGroup with the values given.
    * @param {StopClusterGpuid} stopClusterGpuid - Ground place unique identifier of the stopCluster parent.
-   * @param {StopGroup} stopGroupInfos - StopGroup informations.
-   * @returns {StopGroup}
+   * @param {StopInfos} stopGroupInfos - StopGroup informations.
+   * @returns {void|Error}
    */
   // @ts-ignore
```

Comment:
remove ts-ignore

Reply:
Since the `returns` has changed for a custom `returns`, `@ts-ignore` should be keep here